_docs_use-cases/use-case-consumption-delete-a-third-party-owned-relationshipattribute-and-notify-peer.md

```diff
@@ -19,7 +19,7 @@ properties:
   - tech category: Attributes
   - status: DONE
   - documentation status: DONE
-  - comments:
+  - comments: deprecated use /use-case-consumption-delete-a-thirdpartyrelationshipattribute-and-notify-peer instead
```

Comment:
Add comma after "deprecated" (also within Excel table).

Reply:
We don't have a comma at the associated comment in the code. Also, since this is only used internally, I don't see the need to be too strict here.